### PR TITLE
Various updates for the WebAudio draft charter

### PR DESCRIPTION
**Pull request by plehegar**
- Removed expected completion date for Web MIDI API
- Align horizontal reviews wording with charter template
- Align FTE with our own FTE data (down to 0.1)

@mdjp @rtoy let me know if any of those changes cause concerns.

### CODE REVIEW

File: audio-2019.html [hunk 1]
```diff
@@ -317,14 +315,10 @@ <h3>
...
-              <li>Month YYYY: First teleconference</li>
-              <li>Month YYYY: First face-to-face meeting</li>
-              <li>Month YYYY: Requirements and Use Cases for FooML</li>
-              <li>Month YYYY: FPWD for FooML</li>
-              <li>Month YYYY: Requirements and Use Cases for BarML</li>
-              <li>Month YYYY: FPWD FooML Primer</li>
+            <ul>
+              <li>Q1 2020: Web Audio API 1.0 completed</li>
+              <li>Q1 2022: Web Audio API 2.0 completed</li>
+              <li>Q1 2020: Audio Device Client completed</li>
```
Comment (rtoy) [inline audio-2019.html:321]: 2020 Q1 is probably too agressive.  Q2 or Q3 is probably more realistic.

Comment (plehegar) [reply audio-2019.html:321]: Changed it to Q3 2020

File: audio-2019.html [hunk 2]
```diff
@@ -127,7 +127,7 @@ <h1 id="title">Audio Working Group Charter <span class="todo">DRAFT!</span></h1>
               Team Contacts
             </th>
             <td>
-              <a href="mailto:chris@w3.org">Chris Lilley</a></span> (0.2 <abbr title="Full-Time Equivalent">FTE</abbr>)<br/>
+              <a href="mailto:chris@w3.org">Chris Lilley</a></span> (0.1 <abbr title="Full-Time Equivalent">FTE</abbr>)<br/>
```
Comment (svgeesus) [inline audio-2019.html:130]: Yes, that is what FTEMS shows

File: audio-2019.html [hunk 3]
```diff
@@ -333,7 +327,15 @@ <h3>Timeline</h3>
...
+        <p>For all specifications, this Working Group will seek <a href="https://www.w3.org/Guide/process/charter.html#horizontal-review">horizontal review</a> for
+          accessibility, internationalization, performance, privacy, and security with the relevant Working and
+          Interest Groups, and with the <a href="https://www.w3.org/2001/tag/" title="Technical Architecture Group">TAG</a>.
+          Invitation for review must be issued during each major standards-track document transition, including
+          <a href="https://www.w3.org/Consortium/Process/#RecsWD" title="First Public Working Draft">FPWD</a>.  The
+          Working Group is encouraged to engage collaboratively with the horizontal review groups throughout development of
```
Comment (svgeesus) [inline audio-2019.html:335]: yup, the template had improved since I first drafted this

**Review by svgeesus — APPROVED** [review]
Changes all look good to me

**Pull request merged**